File: app/lib/models/room.dart
class Room{
  int id;
  int rank;
  int cost;
  String resume;
  String largeDescription;
  String img;
  String name;
  int active;

  Room({ this.id, this.rank, this.cost, this.resume, this.largeDescription, this.img, this.name, this.active });

  factory Room.fromJson(Map<String, dynamic> json){
    return Room(
      id: json['id'],
      rank: json['rank'],
      cost: json['cost'],
      resume: json['resume'],
      largeDescription: json['largeDescription'],
      img: json['img'],
      name: json['name'],
      active: json['active']
    );
  }
}

File: laravel/app/Models/room.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class room extends Model
{
    use HasFactory;

    protected $fillable = [
        'rank',
        'cost',
        'resume',
        'large_description',
        'img',
        'name',
        'active'
    ];

    protected $attributes = [
        'img' => null,
        'active' => 1
    ];

    protected $table = 'rooms';

}
